package.json

```diff
@@ -32,17 +32,24 @@
     "tar": "^4.4.8"
   },
   "scripts": {
-    "tailwind:css": "tailwind build src/css/tailwind.src.css -c tailwind.config.js -o src/css/tailwind.css",
-    "start": "npm run tailwind:css && react-scripts start",
-    "build": "npm run tailwind:css && react-scripts build",
+    "tailwind:css": "tailwind build src/css/tailwind.src.pcss -c tailwind.config.js -o src/css/tailwind.css",
+    "start": "npm-run-all --parallel watch:css start:react",
```

Comment:
Does this build component related style sheets (.pcss) too? My setup is the following:
``` 
"scripts": {
    "tailwind:css": "tailwind build src/css/tailwind.src.pcss -c tailwind.js -o src/css/tailwind.css",
    "start": "npm-run-all --parallel watch:css watch:component-css start:react",
    "build": "npm-run-all build:css build:react",
    "build:css": "postcss src/css/tailwind.src.pcss -o src/css/tailwind.css",
    "watch:css": "postcss src/css/tailwind.src.pcss -o src/css/tailwind.css -w",
    "watch:component-css": "postcss src/components/**/*.pcss --base src --dir src --ext module.css -w",
    "start:react": "react-scripts start",
    "build:react": "react-scripts build",
    "predeploy": "npm run build",
    "deploy": "gh-pages -d build",
    "test": "react-scripts test",
    "eject": "react-scripts eject",
    "storybook": "start-storybook -p 9009 -s public",
    "build-storybook": "build-storybook -s public"
  },
```